php/src/App.php

```diff
@@ -13,7 +13,7 @@ public function foo(bool $create): ?Foo
         return null;
     }
 
-    public function bar(Foo $maybeFoo): string
+    public function bar(?Foo $maybeFoo): string
     {
         return $this->buzz($maybeFoo);
```

Comment:
**[psalm]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>error - PossiblyNullArgument: Argument 1 of shmurakami\sandbox\App::buzz cannot be null, possibly null value provided